src/main/resources/db/changelog/data/users.csv

```diff
@@ -0,0 +1,4 @@
+id,user_name,email,password,user_status
+1,user one,user@gmail.com,Password1,verified
```

Comment:
If we assume the complexity of the password, it must contain at the same time:
- small letters
- uppercase
- numbers
- special signs

I think it's better to put passwords that meet this complexity here right away. So my suggestion:

userOnePassword1!
userTwoPassword2!
userThreePassword3!